app/assets/stylesheets/welcome.scss

```diff
@@ -0,0 +1,63 @@
+
+
+video {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  min-width: 102%;
+  min-height: 100%inherit;
+  margin: 50px 0 0 0;
+  width: auto;
+   height: auto;
+   z-index: -100;
+   -ms-transform: translateX(-50%) translateY(-50%);
+   -moz-transform: translateX(-50%) translateY(-50%);
+   -webkit-transform: translateX(-50%) translateY(-50%);
+   transform: translateX(-50%) translateY(-50%);
+   background: url(polina.jpg) no-repeat;
+   background-size: cover;
+}
+
+#welcome-title {
+  font-size: 50px;
+  text-align: center;
+  margin: 30% 0 2% 0;
+}
+.button {
+  @include shadow3;
+  color: #fff;
+  background-color: $green-dark;
+  border: none;
+  border-radius: 5px;
+  margin: 0 0 5px 0;
+  padding: 10px 15px 10px 15px;
+  &:hover {
+    @include shadow2;
+    background-color: $green;
+    // transform: translateY(1px);
+    // box-shadow: 0 4px $green-dark2;
+  }
+  &:active {
+    @include shadow1;
+    background-color: $green-dark1;
+    // transform: translateY(4px);
+    // box-shadow: 0 1px $green-dark2;
+  }
+  &:focus {
+    outline: 0;
+  }
+}
+
+.accounts {
+  text-align: center;
+  margin: 0 0 0 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0` instead of `0 0 0 0`